### PR TITLE
Add rotation command and update help and readme

### LINKED ISSUE #69: Rotation list
Add a list of snack rotations for which team is buying

### PR DESCRIPTION
**Pull request by Lauraducky**
Fixes #69

### CODE REVIEW

**Comment by arthurs** [comment]
should be stored in config otherwise its restrictive to out setup

File: README.md [hunk 1]
```diff
@@ -86,7 +87,8 @@ PockyBot can be messaged directly with certain commands.
 * `welcome` &mdash; display a welcome message.
 * `ping` &mdash; verify that the bot is alive.
 * `help` &mdash; display a list of available commands.
-* `keywords` &mdash; display the available keywords
+* `keywords` &mdash; display the available keywords.
+* `rotation` &mdash; display the rotation.
```
Comment (jimcullenaus) [inline README.md]: What is a 'rotation'? It's not very self-descriptive for someone who comes across this without context.

File: src/lib/response-triggers/help.ts [hunk 2]
```diff
@@ -48,6 +48,9 @@ export default class Help extends Trigger {
 	1. I will respond in the room you messaged me in if I am alive.
 * Welcome someone 👐!
 	1. To get a welcome message from me, type \`@${constants.botName} welcome\` OR direct message me with \`welcome\`.
+	1. I will respond in the room you messaged me in.
+* Check the rotation!
+	1. To check the rotation, type \`@${constants.botName} rotation\` OR direct message me with \`rotation\`.
```
Comment (jimcullenaus) [inline src/lib/response-triggers/help.ts]: This also needs further explanation or context.

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 FROM node:9-alpine
-LABEL version "1.3.1"
+LABEL version "1.4.1"
```
Comment (drewfreyling) [inline Dockerfile]: this is an invalid label. should be in the format `LABEL version="1.4.1"`

**Pull request merged**